src/app/language.js

```diff
@@ -23,6 +23,12 @@ var setLanguage = function (preferredLanguage) {
     });
 };
 
+
+App.Localization.nativeName = function (lang) {
+    var codes = App.Localization.langcodes
+    return codes[lang]?codes[lang].nativeName:'No native Name for: '+lang;
+}
```

Comment:
This doesn't pass jshint tests (missing semicolon and spaces). I also wonder if the condition are good, I mean, it's not because you have codes[lang] that you automatically have codes[lang].nativeName. I'm also wondering if you shouldn't try to solve the issues elsewhere instead of sending back a "error" message. How about:
```
App.Localization.nativeName = function (lang) {
    var codes = App.Localization.langcodes;
    return codes[lang] && codes[lang].nativeName ? codes[lang].nativeName : null;
};```